test/integration/suite_test.go

```diff
@@ -214,7 +214,6 @@ func deployControllers(ctx context.Context, ready chan ktfkind.ProxyReadinessEve
 			flags.Parse([]string{
 				fmt.Sprintf("--kong-admin-url=http://%s:8001", adminHost),
 				fmt.Sprintf("--kubeconfig=%s", kubeconfig.Name()),
-				"--proxy-sync-seconds=0.05", // run the test updates at 50ms for high speed
```

Comment:
I actually found that in postgres DB mode, this _appeared to be causing flakes_ though it worked fine in dbless.

I've started a follow-up conversation about this: https://app.slack.com/client/T0DS5NB27/C011RQPHDC7/thread/C011RQPHDC7-1621449050.111500

But for now, working around it by just using the defaults. There may be an upstream bug at play here that we'll have to tease out, but if there is I have full confidence this PR isn't involved, it's just circumstantial.